documentation/Using-the-SDK/Building-a-table.md

```diff
@@ -102,7 +102,7 @@ tableBuilder.SetDefaultTableConfiguration(tableConfig);
 
 ### ColumnRole
 
-A `ColumnRole` is metadata information about a column that defines the column's role in data presentation. For example, a column that contains a `Timestamp` whose value indicates when an event occured relative to the start of a `DataSource`](#datasource)` may be marked as a "start time" Column:
+A `ColumnRole` is metadata information about a column that defines the column's role in data presentation. For example, a column of `Timestamp` values relative to the start of a `DataSource` may be marked as a "start time" Column:
```

Comment:
I duplicated this definition from the glossary - can you update it there to match?